python/requirements.in

```diff
@@ -1,4 +1,9 @@
+architect-functions
 astropy
+boto3
+cachetools
+email-validator
 fastapi
 mangum
 requests
+types-cachetools
```

Comment:
Technically should this go in the root requirements.txt file, as it is only a dependency for development?

Reply:
Fixed in 57c8237.